app/stylesheets/Free.scss

```diff
@@ -1,4 +1,4 @@
-.tilt-icon.tilt-icon-free {
+.tilt-icon.tilt-icon-tag-free {
```

Comment:
is this change needed? we just added the `Free` icon earlier today, but this could potentially be a breaking change.